test/dash/mpd_utils_unit.js

```diff
@@ -785,8 +768,8 @@ describe('MpdUtils', () => {
     }
 
     function testRequest(baseXMLString) {
-      const xml = parser.parseFromString(baseXMLString, 'text/xml')
-          .documentElement;
+      const xml = /** @type {shaka.extern.xml.Node} */ (
+        shaka.util.TXml.parseXmlString(baseXMLString));
```

Comment:
nit: add two spaces in the indentation

Reply:
The linter reverts to this spacing unfortunately.